Resources/public/js/views/dashboard/ez-dashboardblockmydraftsview.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockmydraftsview', function (Y) {
+    'use strict';
+
+    /**
+     * Provides the My Drafts Dashboard Block View class
+     *
+     * @module ez-dashboardblockmydraftstview
+     */
+    Y.namespace('eZ');
+
+    var BLOCK_IDENTIFIER = 'my-drafts';
+
+    /**
+     * The my drafts dashboard block view
+     *
+     * @namespace eZ
+     * @class DashboardBlockMyDraftsView
+     * @constructor
+     * @extends eZ.DashboardBlockAsynchronousView
+     */
+    Y.eZ.DashboardBlockMyDraftsView = Y.Base.create('dashboardBlockMyDraftsView', Y.eZ.DashboardBlockAsynchronousView, [], {
+        initializer: function () {
+            this._set('identifier', BLOCK_IDENTIFIER);
+        },
+
+        /**
+         * Gets item template data.
+         *
+         * @method _getTemplateItem
+         * @protected
+         * @param item {Y.Model} draft item model
+         */
+        _getTemplateItem: function (draft) {
+            return {
+                version: draft.version.toJSON(),
+                contentType: draft.contentType.toJSON(),
+                contentInfo: draft.contentInfo.toJSON()
+            };
+        },
+
+        /**
+         * Makes request for data
+         *
+         * @method _getDraftsData
```

Comment:
wrong name, also the description should be a bit more precise since it's a reimplementation of a parent class method.